src/main/scala/DataMunging.scala

```diff
@@ -0,0 +1,40 @@
+package DataMunging
+import scala.io.Source
+
+object DataMunging {
+
+    def main(args: Array[String]) ={
+            getMinSpread()
+            getMinGoalDiff()
+    }
+    def getMinSpread() = {
+        val source = Source.fromFile("weather.dat")
+        val spread = source.getLines.toList
+                               .drop(2) //the first line is a header and second line is a blank space so skip those lines here.
+                               .map(_.split("\\s+")) // split on regex white space
+                               .map(x => (x(1), x(2).stripSuffix("*").toDouble-x(3).stripSuffix("*").toDouble)) //clean the recourds as somelines have * and aslo conver to double
+        val min_spread = find_min(spread) // get minimum spread
```

Comment:
Scala style guide usually goes with camelCase variable names like minSpread, rather than having underscores like min_spread 

http://docs.scala-lang.org/style/